workbench-core/audit/src/baseExtractor.ts

```diff
@@ -33,10 +33,14 @@ export const BaseExtractor: Extractor = {
       };
 
       // Sets actor to be the AuthenticatedUser's id.
-      const authenticatedUser: AuthenticatedUser = retrieveUser(res);
-      const actor = {
-        uid: authenticatedUser.id
-      };
+      let actor = { uid: 'user not found' };
+      try {
+        const authenticatedUser: AuthenticatedUser = retrieveUser(res);
```

Comment:
Should we provide `isAuthenticated` functionality?

Reply:
Added this request to `MAFoundation-545`